azurelinuxagent/common/protocol/hostplugin.py

```diff
@@ -480,8 +480,9 @@ def format_message(msg):
                 logger.info(message)
                 add_event(op=WALAEventOperation.HostPlugin, message=message, is_success=True)
 
-            # Don't support HostGAPlugin versions older than 115
-            if vm_settings.host_ga_plugin_version < FlexibleVersion("1.0.8.115"):
+            # Don't support HostGAPlugin versions older than 123
+            # TODO: update the minimum version to 1.0.8.117 before release
```

Comment:
Just to clarify, you mean minimum version to 123 before release?

Reply:
yes, i'll fix the comment